CI/travis.osx.install.sh

```diff
@@ -38,9 +38,7 @@ for i in $BREWS; do
   for RETRIES in $(seq 1 3); do
     echo " "
     echo "Installing ${i}"
-    #Added the -w (whole-word) option so that the grep will NOT match for pcre2
-    #when we are considering pcre:
-    brew list | grep -w -q $i || brew install $i
+    brew install "$i"
```

Comment:
Former brew versions used to complain if a lib was already installed which is why we had this construct. Is that not the case anymore?

Reply:
It seems to be happy now:

```
% brew install qt@5
==> Downloading https://formulae.brew.sh/api/formula.jws.json
######################################################################### 100.0%
==> Downloading https://formulae.brew.sh/api/cask.jws.json
######################################################################### 100.0%
Warning: qt@5 5.15.12 is already installed and up-to-date.
To reinstall 5.15.12, run:
  brew reinstall qt@5
```

While piping brew list doesn't work anymore:

```
% brew list | echo

Error: Failure while executing; `ls /opt/homebrew/Cellar` was terminated by uncaught signal PIPE.
(base) vadim.peretokin@Vadim-MBP ~ %
```

Reply:
What was the status value `$?` of that brew install with an already installed package?

Reply:
:shipit: